core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -422,14 +461,23 @@ private[kafka] abstract class AbstractServerThread(connectionQuotas: ConnectionQ
   def wakeup(): Unit
 
   /**
-   * Initiates a graceful shutdown by signaling to stop and waiting for the shutdown to complete
+   * Initiates a graceful shutdown by signaling to stop
    */
   def shutdown(): Unit = {
```

Comment:
Should we rename this method to be `initiateShutdown()` to be consistent with `kafka.utils.ShutdownableThread`?